src/views/GameDetailView.vue

```diff
@@ -0,0 +1,73 @@
+<script setup>
+import { ref, onMounted } from 'vue';
+import { useGameLibraryStore } from '../stores/GameLibraryStore.js'; 
+import { useRoute } from 'vue-router';
+
+const route = useRoute();
+
+const gameLibraryStore = useGameLibraryStore();
+
+if(!gameLibraryStore.count) {
+  await gameLibraryStore.getGameDetails(route.params.id);
+}
+</script>
+
+<template>
+  <main class="game-detail-view">
+    <h1>Game Details</h1>
+
+    <h2>{{ gameLibraryStore.gameDetails.name }}</h2>
+
+    <img :src="gameLibraryStore.gameDetails.imageUrl" loading="lazy" />
+
+    <p class="game-description">{{ gameLibraryStore.gameDetails.description }}</p>
```

Comment:
To display the escaped characters in the description, we should use `v-html`

`<p class="game-description" v-html="gameLibraryStore.gameDetails.description"></p>`